Accessories/ExpansionBoard_uInverter/REV20210506A/uInverter-rev-a_bom.csv

```diff
@@ -1,20 +1,20 @@
 Supplier Part Number 1,Quantity,Comment,Description,Designator,Footprint,LibRef
```

Comment:
Is there a reason the REV A BOM is changed in this PR?

Reply:
Yes, the description in docs and the BOM itself mentions a 2.2K resistor, however the part number mentioned was for 2.49K, so I corrected that and replaced it with a 2.2K part no. (same 0603 size)

Reply:
1. Units for both, drill files and Gerber files are inches
2. I did not add the board outline before. Now, I have added the outline on `keepout layer` and pushed the changes.

Reply:
@purohitparth25 Sounds good for the REV A BOM change, I trust this change was propagated to the REV B BOM as well.

Reply:
@npetersen2, we are using an altogether different part number in revB (0603 size as compared to 0805 in revA)

Reply:
@purohitparth25 Excellent, thanks!